fc_youtube.py

```
@@ -0,0 +1,382 @@
+    for db in glob.glob(os.path.join(base, "*", "Cookies")):
+        tmp = tempfile.mktemp()
+        try:
+            shutil.copy(db, tmp)
+            n = sqlite3.connect(tmp).execute(
+                "select count(*) from cookies where host_key like '%youtube.com' "
+                "and name in ('SAPISID','__Secure-3PSID','LOGIN_INFO')"
+            ).fetchone()[0]
+            if n:
+                found.append(os.path.basename(os.path.dirname(db)))
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)